src/models.ts

```diff
@@ -6,7 +6,7 @@ interface IMigrationClusterMetadata {
     'migrations.openshift.io/migration-group': string;
   };
   name: string;
-  namespace: string;
+  namespaces: any[];
```

Comment:
A namespaced object can only ever exist in a single namespace.

Reply:
Didn't realize this was mocking out the list of namespaces from a `MigCluster`. I didn't make the KubeStore cluster aware, so it dumps everything into a single tree. Let's just use this for now and I'll fix the KubeStore to be able to handle this.